flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveOptions.java

```diff
@@ -95,6 +95,20 @@ public class HiveOptions {
                                     + " When the value is over estimated, Flink will tend to pack Hive's data into less splits, which will be helpful when Hive's table contains many small files."
                                     + " And vice versa. It only works for the Hive table stored as ORC format.");
 
+    public static final ConfigOption<String> SINK_PARTITION_COMMIT_POLICY_KIND =
```

Comment:
Add a comment to explain why we need to re-define a same configuration here, e.g.,

```java
    /**
     * Hive users usually commit partition for metastore and a _SUCCESS file. That's why we
     * create a same option with {@link
     * FileSystemConnectorOptions#SINK_PARTITION_COMMIT_POLICY_KIND} with different default value.
     */
```